introspection/nm-manager.xml

```diff
@@ -381,6 +381,19 @@
     <property name="Version" type="s" access="read"/>
 
     <!--
+        Capabilities:
+
+        The current set of capabilities.  This is a list of 32 bit
+        integers where specific bits are set when a capability is
+        currently available.  For example, capability number 42 would
+        be represented by bit (42 % 32) == 10 in the element with index
+        (42 / 32) == 1 in the array.
+
+        See <link linkend="NMCapability">NMCapability</link> for currently defined capability numbers.
+    -->
+    <property name="Capabilities" type="au" access="read"/>
```

Comment:
>  I like the idea of using an array of u32 where each element is a capability which can assume any value.

I agree.  I used a bitfield because the code was simpler.  If everybody agrees, let's use a set of integers.  Should they be guaranteed to be sorted?  I think that would be nice.